release/downloadIstioCtl.sh

```diff
@@ -0,0 +1,50 @@
+#!/usr/bin/env bash
+#
+# Separate downloader for istioctl
+#
+# This file will be fetched as: curl -L https://git.io/getLatestIstioCtl | sh -
+#
+detect_os() {
+  OS=$(uname)
+  case $OS in
+    'Linux')
+      OSEXT='linux'
+      FILENAME='istioctl'
+      ;;
+    'WindowsNT')
+      OSEXT='win.exe'
+      FILENAME='istioctl.exe'
+      ;;
+    'Darwin') 
+      OSEXT='osx'
+      FILENAME='istioctl'
+      ;;
+    *) ;;
+  esac
+}
+
+get_version() {
+  if [ "x${ISTIO_VERSION}" = "x" ] ; then
+  ISTIO_VERSION=$(curl -L -s https://api.github.com/repos/istio/istio/releases/latest | \
+                  grep tag_name | sed "s/ *\"tag_name\": *\"\\(.*\\)\",*/\\1/")
+  fi
+}
+
+download_istioctl() {
+  NAME="istioctl-${OSEXT}"
+  URL="https://storage.googleapis.com/istio-artifacts/${ISTIO_VERSION}/artifacts/istioctl/${NAME}"
+  echo "${URL}"
+  echo "Downloading ${NAME} ..."  
```

Comment:
can these two echo lines be merged into 1?

Reply:
Yes, they can be.